src/api/auth.ts

```diff
@@ -88,6 +89,16 @@ async function findOrCreateUser(clientId: string, email: string): Promise<User>
     const roles: [Role] = [isAdmin ? 'ADMIN' : 'VISITOR'];
     console.log(`Creating user ${email} roles ${roles} id ${clientId}`);
     try {
+      // using sendgrid to notify admin a new user is created
```

Comment:
1. move this block to outside of and _before_ this try block. The try block is for User.create() only. Naturally email sending should be after user is created in db, but logically only the reverse order is error proof. (I can explain further if needed.)

2. create a new function in sendgrid.ts called sendEmailToAdmin() which only accept one parameter which is the message to be included in the email. So here you can simply call `sendEmailToAdmin(`New user ${email} signed up`)`